src/main.tf

```diff
@@ -8,7 +8,10 @@ terraform {
       source  = "hashicorp/azuread"
       version = "= 2.6.0"
     }
-
+    postgresql = {
+      source  = "cyrilgdn/postgresql"
```

Comment:
is it safe to use this provider? Is it kind of ufficial?

Reply:
I removed this provider and we decided to create user/grant postegresql via flyway script